CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+ advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+ address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+ professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at guilherme.gois@gorilaivnest.com.br. All
+complaints will be reviewed and investigated and will result in a response that
```

Comment:
"will be reviewed and investigated" by whom and how? What are the procedures? How can we assure that all parties will be treated fairly? What defines "appropriate"? Honestly, I recognize and fully support the good intentions behind this, but I think the inclusion of this document in the repository may create more problems than it solves. Do we really need this?

Reply:
As in [Open Source Guide](https://opensource.guide/code-of-conduct/)
> A code of conduct is a document that establishes expectations for behavior for your project’s participants. Adopting, and enforcing, a code of conduct can help create a positive social atmosphere for your community.

I don't see why we shouldn't have a code of conduct, also having and enforcing one, always developing an environment where people still confortable to stay and interact, fixing bugs, raising discussions, developing or requiring new features, just show how mature and solid is the project

Reply:
As mentioned above, whatever is in this document could have an impact outside of the project and deeply affect someone's career and personal life. There are too many questions and undefined aspects and details here.

Reply:
I think that is a good thing to think and discuss. However, It needs a continuous effort of everyone, specially of managers. Create laws without fiscalization and punishment is the same thing of don't have laws.